### PR TITLE
add amount to result in fillTransactionStatus

### PR DESCRIPTION
**Pull request by valpinkman**

### CODE REVIEW

File: src/families/ethereum/modules/erc20.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ const erc20approve: ModeModule = {
     if (!t.useAllAmount && t.amount.eq(0)) {
       result.errors.amount = new AmountRequired();
     }
+    result.amount = t.useAllAmount ? BigNumber(2).pow(256).minus(1) : t.amount;
```
Comment (LFBarreto) [inline src/families/ethereum/modules/erc20.js:41]: just use t.amount here the UI handles the no limit

File: src/families/ethereum/modules/compound.js [hunk 2]
```diff
@@ -70,6 +70,7 @@ const compoundSupply: ModeModule = {
     } else if (t.amount.gt(subAccount.spendableBalance)) {
       result.errors.amount = new NotEnoughBalance();
     }
+    result.amount = t.useAllAmount ? a.balance : t.amount;
```
Comment (LFBarreto) [inline src/families/ethereum/modules/compound.js:73]: i also think using t.amount here is also safe as is for withdraw and enable

File: src/families/ethereum/modules/erc20.js (hunk 1 above)
Comment (gre) [reply src/families/ethereum/modules/erc20.js:41]: `BigNumber(2).pow(256).minus(1)` isn't it a constant somewhere?

Comment (valpinkman) [reply src/families/ethereum/modules/erc20.js:41]: I think we haven't set it up but I will

**Review by LFBarreto — APPROVED** [review]
:+1:

**Review by gre — APPROVED** [review]

**Pull request merged**